support-frontend/assets/pages/[countryGroupId]/guardianAdLiteLanding/components/posterComponent.tsx

```diff
@@ -73,6 +74,16 @@ const copy = css`
 const paragraph = css`
 	margin-bottom: ${space[3]}px;
 `;
+const displayMobile = css`
+	${from.desktop} {
```

Comment:
Isn't `from.desktop` desktop and larger? Just trying to figure out how that fits with `displayMobile`

Reply:
`display: none` underneath means its reversed, can write it in a positive way instead, have updated to that now if you prefer.

Reply:
Ah sorry I must have not drunk enough coffee when I wrote this! I'm happy with either approach.